Microsoft.Toolkit.Uwp.UI/Helpers/Emoji/EmojiManager.cs

```diff
@@ -0,0 +1,391 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.Toolkit.Uwp.UI.Helpers.Emoji
+{
+    /// <summary>
+    ///     Class to help manage use of Emojis
+    /// </summary>
+    public static class EmojiManager
+    {
+        private static readonly EmojiDetails[] EmojiArray =
+        {
+            new EmojiDetails(Emoji.HelmetWithWhiteCross, new[] {0x26D1, 0xFE0F}, true), new EmojiDetails(Emoji.Kiss, new[] {0x1F48F}, true), new EmojiDetails(Emoji.CoupleWithHeart, new[] {0x1F491}, true),
```

Comment:
If new emojis get added, I'm sure someone will raise an issue and we can amend our list and add it in. Now that we have an initial list of emojis, adding in a few extra ones won't be difficult. I can always just post the generator code I used to my github I suppose in case anyone did want to use that to just regenerate the lot.